configure

```diff
@@ -1928,7 +1934,9 @@ struct stat;
 /* Most of the following tests are stolen from RCS 5.7 src/conf.sh.  */
 struct buf { int x; };
 struct buf * (*rcsopen) (struct buf *, struct stat *, int);
-static char *e (char **p, int i)
+static char *e (p, i)
+     char **p;
+     int i;
```

Comment:
Is this correct? It doesn't look like valid C code?

Reply:
That's generated code by `autoconf`.